apps/user-office-frontend/src/components/proposal/ProposalSummary.tsx

```diff
@@ -50,6 +50,9 @@ function ProposalReview({ confirm }: ProposalSummaryProps) {
 
   const proposal = state.proposal;
 
+  // Declare state of submission. This is first submission. (3-Feb-23)

```

Comment:
Looks fine but l don't think we need this comment the code is self explanatory.

Reply:
Updated. Please check, thanks.